oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -126,6 +126,7 @@ ADRs
      - Inspired by `Nygard's post`_, Architecture Decision Records (ADRs) are local lightweight documents of technical decisions co-located with their corresponding code. They provide rationale behind the code and the archeological path to their current state. The status of an ADR can be:
 
        * **draft** if the decision is still preliminary and in experimental phase
+       * **provisional** if the decision is still preliminary and in experimental phase
```

Comment:
The descriptions for "draft" and "provisional" say the same thing, which is confusing.

I don't think we need a "draft" status at all. If your ADR is being proposed in a PR, then it is obvious that it's a draft/proposal. I have never looked at an open PR for a new ADR where the status is "accepted" and thought, "this ADR is already accepted!". 

IMO the simplest way to solve this ambiguity would be to just have ADR authors populate the Status section with the status that they hope to achieve upon merging the PR. That is, if my ADR is to be accepted upon merging of the PR, then I would just write "accepted" into the status section. Same deal for "provisional".

(I wrote ADR above, but I feel the same for OEPs)

Reply:
1. Thanks. I missed copying the updated Draft description from the template below, which is: "if the decision is newly proposed and in active discussion". I will update.
2. I think doing away with the Draft (or Under Review) state is a bigger change than I am proposing. Some people use this status in the PR to provide a bit more of a feel of the readiness of the proposal. Others open the PR in a Draft state, and others have used "[Draft]" in the subject. Also, I think the original intention was that a "Draft" OEP could be merged, which we haven't done much. If you are looking to solve this issue, could you propose a separate PR for that? I know it is related, but it feels like a bigger discussion that I want to avoid. I'm looking for a quick win.
3. That said, I am open to recommending that plain old "Provisional" or "Accepted" could **also** be used when a PR is under review, rather than "Draft" or "Under Review", to state the status that is intended after being merged. Thoughts?

Reply:
@kdmccormick: What do you think of the latest wording?: https://github.com/openedx/open-edx-proposals/pull/344/commits/1f5172f61555735c617a8938010af2945b5e6558

Reply:
Looks great, thanks!